contracts/bridges/arbitrum/ArbitrumBridgeReceiver.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../BaseBridgeReceiver.sol";
+import "./AddressAliasHelper.sol";
+
+contract ArbitrumBridgeReceiver is BaseBridgeReceiver {
+    /**
+     * @notice Receive bridged message and enqueue in the Timelock
+     * @param data ABI-encoded data of the bridged message
+     */
+    function processMessageFromRoot(
```

Comment:
I've implemented this as a `processMessageFromRoot` function, but we could probably make this simpler by just writing a `fallback` function like the `OptimismBridgeReceiver`:

https://github.com/compound-finance/comet/pull/613/files#diff-547cb2f83c28a0936c92c55285d70612e9ea223122a445fe08136684e40c48c0R25

Reply:
updated